src/WebJobs.Script.WebHost/Controllers/SwaggerController.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Microsoft.Azure.WebJobs.Script.WebHost.Filters;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Controllers
+{
+    [AuthorizationLevel(AuthorizationLevel.Function, ScriptConstants.DocumentationResourceName)]
+    public class SwaggerController : ApiController
+    {
+        private readonly ISwaggerDocumentManager _swaggerDocumentManager;
+        private readonly WebScriptHostManager _scriptHostManager;
+
+        public SwaggerController(ISwaggerDocumentManager swaggerDocumentManager, WebScriptHostManager scriptHostManager)
+        {
+            _swaggerDocumentManager = swaggerDocumentManager;
+            _scriptHostManager = scriptHostManager;
+        }
+
+        [HttpGet]
+        [Route("admin/host/swagger/default")]
+        public IHttpActionResult GetGeneratedSwaggerDocument()
+        {
+            var swaggerDocument = _swaggerDocumentManager.GenerateSwaggerDocument(_scriptHostManager.HttpRoutes);
```

Comment:
Rather than swallow exceptions, I think you should catch here and return a 500 if it fails, rather than return 200 in error cases.